docs/interview/02-CSS/02-medium/02-medium.md

```diff
@@ -2,5 +2,120 @@
 
 <details>
   <summary>Explain Inline, Block and Inline Block property in CSS?</summary>
```

Comment:
explain it with respect to CSS. 
for example, we can explicitly change the `inline`, `block`, and `inline-block` behavior of HTML elements using CSS.